### ISSUE TITLE
List of PIConGPU papers

### ISSUE THREAD

**Issue by PrometheusPi** [ComputationalRadiationPhysics/picongpu#3258]
In #3223 cbontoiu brought up the idea of providing a list of PIConGPU related publications online. @ComputationalRadiationPhysics/picongpu-developers What do you think of such a list, maintained e.g. on the ReadTheDocs page. Please vote :+1: if you agree and :-1: if you oppose such a list. 

If we agree on such list, I will prepare a `*.rst` page.

**Comment by Anton-Le** [comment]
I would add that the list should probably be structured and not just a (alphabetical) dump of all that has been written, s.t. a user that is sceptical of the performed testing/validation could easily find the publication that would pacify their misgivings (or could confidently point us to the likely problem).

**Comment by PrometheusPi** [comment]
@Anton-Le Good point. Considering your input, I would suggest a topic specific selection that references a chronologically ordered list. Thus paper with application in various fields as e.g. @bussmann's SC'13 paper - which has relevance in HPC, and astrophysics.

**Comment by ax3l** [comment]
Great idea, I would recommend to extend the list in the reference section and add alongside a recommended "recent code citation" all related topics such as developed plugins.

Suggested order for index of papers, own as well as implemented works:
- general code references (currently SC13; my thesis; soon PoP special issue paper; older PIConGPU papers)
- HPC kernels: SC13, Alpaka papers, libGPUGrid (now: PMacc)
- ionization-physics: MA thesis Marco
- QED work: e.g. MA thesis Heiko; new plugins scattering; pusher RR
- diagnostics: Richard's papers; Voronoi merging
- field solvers: implemented papers (others)
- pushers: implemented papers (others)

I tried to add citations to some extend already in our release meta data, please take and add to these as well: https://github.com/ComputationalRadiationPhysics/picongpu/blob/659866c5ad2f56b2acc600f3155bb59e0f417f15/.zenodo.json#L113-L124

We also scatter these references over the manual already, but a dedicated page again order by topics and maybe with comments (e.g. "implementation aspect X is up to date/outdated") would be fantastic.

**Comment by cbontoiu** [comment]
I would add an item in @ax3l list dedicated to ```applications``` for those papers that don't contribute to the code but only use it to simulate various physical situations. These papers could be the first contact for unexperienced people, who are looking for a suitable code and will show what they could do with PIConGPU.

**Comment by PrometheusPi** [comment]
I agree with @cbontoiu. It would also be very beneficial to have a list of papers that present use cases for PIConGPU (without code contributions) that is a good reference what can be achieved with PIConGPU.

**Comment by Anton-Le** [comment]
@ax3l For references that have been aggregated to this list from other documentation pages I think it would be meaningful to add a reference/note (or link) to the specific documentation pages. 

In this way when looking through the aggregated list users should be able to easily find documentation pages relevant to them without trying to dig through the entire documentation.

**Comment by PrometheusPi** [comment]
At everyone: thanks for participating in the vote. 

Everyone that contributed favors such a list. I will prepare a version and will try to include all your ideas. We can discuss details during the review process of the pull request.

**Comment by cbontoiu** [comment]
I don't know if here or elsewhere we could add a list of published papers from the users of PIConGPU. Here is one from myself and my group https://iopscience.iop.org/article/10.1088/1742-6596/1596/1/012028/meta

**Comment by cbontoiu** [comment]
This is dedicated to you, with all my admiration for your work. I am deeply grateful for your support!

https://www.nature.com/articles/s41598-023-28617-w

**Comment by PrometheusPi** [comment]
Thanks @cbontoiu for informing us. Both your papers are [now included](#4445) in our reference list.

**Comment by PrometheusPi** [comment]
Reminder to myself: 
need to add BA/MA thesis from 4 students

**Comment by BrianMarre** [comment]
@PrometheusPi an update?
